### PR TITLE
[Kiali] issue #3348 update kiali task doc so it mentions the new default secret

### PR DESCRIPTION
**Pull request by jmazzitelli**
addresses issue https://github.com/istio/istio.io/issues/3348

### CODE REVIEW

File: content/docs/tasks/telemetry/kiali/index.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ The following instructions assume you have installed Helm and use it to install
 To install Kiali without using Helm, follow the [Kiali installation instructions](https://www.kiali.io/gettingstarted/).
 {{< /idea >}}
 
+### Create a Secret
```
Comment (geeknoid) [inline content/docs/tasks/telemetry/kiali/index.md]: Secret -> secret

File: content/docs/tasks/telemetry/kiali/index.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ The following instructions assume you have installed Helm and use it to install
...
+
+{{< idea >}}
+If you plan on installing Kiali using the `istio-demo.yaml` or `istio-demo-auth.yaml` file as described in the [Istio Quick Start Installation Steps](https://istio.io/docs/setup/kubernetes/quick-start/#installation-steps) then a default secret will be created for you with a username of `admin` and passphrase of `admin`. You can therefore skip this section.
```
Comment (geeknoid) [inline content/docs/tasks/telemetry/kiali/index.md]: Drop the 'https://istio.io' from the URL.

File: content/docs/tasks/telemetry/kiali/index.md (hunk 1 above)
Comment (jmazzitelli) [reply content/docs/tasks/telemetry/kiali/index.md]: done.

File: content/docs/tasks/telemetry/kiali/index.md (hunk 2 above)
Comment (jmazzitelli) [reply content/docs/tasks/telemetry/kiali/index.md]: done.

**Pull request merged**